src/components/dropdown/Dropdown.tsx

```diff
@@ -391,6 +399,7 @@ export class Dropdown<T = LabelValue> extends React.Component<
     isTypeAheadEnabled: true,
     autoScrollToCurrent: false,
     enableTermSearch: false,
+    menuShouldScrollIntoView: true,
```

Comment:
by default as true?

Reply:
The default value in react-select is:

`menuShouldScrollIntoView boolean = !isMobileDevice()`

which resolves to true on desktop. Do we want to change default to !isMobileDevice()? Not complicated with Client Hints (https://web.dev/user-agent-client-hints/) and then maybe fallback to true?